lib/str.ts

```diff
@@ -1070,36 +1003,23 @@ const Str = {
         return website.toLowerCase() + this.cutBefore(match[1], match[2]);
     },
 
-    /**
-     * Checks if parameter is a string or function
-     * if it is a function then we will call it with
-     * any additional arguments.
```

Comment:
Maybe we can leave the comment here as well so it's shown to the dev during usage